.github/workflows/build.yml

```diff
@@ -6,6 +6,10 @@ on:
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.os }}
```

Comment:
Is `matrix.os` resolvable here? Or should it be the runner's OS?

Reply:
https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs says it is